### ISSUE TITLE
kompose up/down not creating/deleting POD object generated with convert

### ISSUE THREAD

**Issue by surajssd** [kubernetes-incubator/kompose#342]

**Comment by cdrage** [comment]
Hey @surajssd do you mind giving a test example? I'm trying to replicate this.

**Comment by kadel** [comment]
what this is about?

**Comment by kadel** [comment]
@surajssd 
Is this issue about this?

```yaml
version: "2"

services:
  foo:
    image: "busybox"
    restart: "on-failure"
    command: ["sleep", "30s"]
```

```
▶ kompose up       
WARN[0000] [foo] Service cannot be created because of missing port. 
We are going to create Kubernetes Deployments, Services and PersistentVolumeClaims for your Dockerized application. 
If you need different kind of resources, use the 'kompose convert' and 'kubectl create -f' commands instead. 

FATA[0000] Error while deploying application: Deployment.extensions "foo" is invalid: spec.template.spec.restartPolicy: Unsupported value: "OnFailure": supported values: Always 
```

**Comment by surajssd** [comment]
here is the docker-compose file i am using
```yaml
$ cat docker-compose.yml 
version: "2"

services:
  mariadb:
    image: centos/mariadb
    restart: "no"
```


when doing convert
```bash
$ kompose convert
INFO[0000] file "mariadb-pod.json" created
```

when doing up nothing is created
```bash
$ kompose up                                                                                                                                                               
We are going to create Kubernetes Deployments, Services and PersistentVolumeClaims for your Dockerized application. 
If you need different kind of resources, use the 'kompose convert' and 'kubectl create -f' commands instead. 


Your application has been deployed to Kubernetes. You can run 'kubectl get deployment,svc,pods,pvc' for details.
```

sorry for incomplete info above.

**Comment by kadel** [comment]
fixed in #363
